### ISSUE TITLE
Gerber file discovery win 10

### ISSUE THREAD

**Issue by mikhailmihalkov** [gerbv/gerbv#154]
If the gerbv program is assigned by default, the gerber file cannot be opened (empty window)... File opens only after starting the program (menu open) win 10\64

**Comment by ooxi** [comment]
@mikhailmihalkov which file extensions are you associating with gerbv? A gerbv project file or gerber layer files themselves?

**Comment by ooxi** [comment]
Similar to issue #107

**Comment by mikhailmihalkov** [comment]
> @mikhailmihalkovкакие расширения файлов вы связываете с gerbv? Файл проекта gerbv или сами файлы слоя gerber?

I assign to all types that generate cad and, when saved by the program itself, does not open gerbv in the same way.... gbr.

**Comment by mikhailmihalkov** [comment]
> @mikhailmihalkovкакие файлы расширения вы связываете с gerbv? Файл проекта gerbv или сами файлы слоя gerber?

Yes, I make an appointment to open with....

**Comment by mikhailmihalkov** [comment]
> Аналогично выпуску [№107](https://github.com/gerbv/gerbv/issues/107)

A similar problem is not solved... Did not see it when opening the request)

**Comment by gyachts** [comment]
> Similar to issue #107

I would be interested in adding the feature to Windows builds to detect a project file (.gvp ext) so Windows can start gerbv with the project loaded without the command line switches of '--project=<prjctfile'.  I currently solved it using a Power Script.
Do you want me to make a new PR or use #107?

**Comment by spe-ciellt** [comment]
New packaging, please test after release of 2.13.0 tomorrow. Closing this for now.
